README.md

```diff
@@ -162,6 +145,34 @@ This command has two modes: bounded and unbounded.
 ##### Unbounded
 If only a start ledger is provided, then the command runs in an unbounded fashion starting from the provided ledger. In this mode, the Stellar Core connects to the Stellar network and processes new orderbooks as they occur on the network. Since the changes are continually exported in batches, this process can be continually run in the background in order to avoid the overhead of closing and starting new Stellar Core instances.
 
+#### export_transactions
+
+```bash
+> stellar-etl export_transactions --start-ledger 1000 \
+--end-ledger 500000 --output exported_transactions.txt
+```
+
+This command exports transactions within the provided range.
+
+#### export_operations
+
+```bash
+> stellar-etl export_operations --start-ledger 1000 \
+--end-ledger 500000 --output exported_operations.txt
+```
+
+This command exports operations within the provided range.
+
+#### export_effects
+
+```bash
+> stellar-etl export_operations --start-ledger 1000 \
+--end-ledger 500000 --output exported_effects.txt
+```
+
+This command exports effects within the provided range.
```

Comment:
Why did this block move? It seems like there's still an `--end-ledger`, meaning it's not an unbounded range like the new heading implies.

Reply:
These commands were in History Archive section before that's why I moved them. To make it more clear, I added a "Stellar Core bounded mode only" at [this commit](https://github.com/stellar/stellar-etl/pull/168/commits/f953baf995b631c495137e8d2381485ae1e45fff).